machine/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
+  - JoelSpeed
+  - Fedosin
+  - elmiko
+  - lobziik
+  - alexander-demichev
+  - mandre
+  - mdbooth
```

Comment:
```suggestion
```

Reply:
I don't think they should be there.

Reply:
they plan to add openstack providerspec here and maintain it.

Reply:
I'd suggest keeping the list reasonable max. 2-3 names not all team members.